src/Collection/ProcessArgumentsCollection.php

```diff
@@ -150,4 +151,19 @@ public function addArgumentWithCommaSeparatedFiles($argument, FilesCollection $f
 
         $this->add(sprintf($argument, implode(',', $paths)));
     }
+
+    /**
+     * @return string
+     */
+    public function generateCliCommand()
+    {
+        $commandlineArgs = $this->getValues();
+
+        $commandline = implode(' ', array_map(
+            [ProcessUtils::class, 'escapeArgument'],
```

Comment:
please don't call methods by name. it's super hard to perform SCA on code if you hide method name under strings

Reply:
Okay, I will change this.

Styling wise, do you guys prefer **`array_map` with a callback**:

```
$commandlineArgs = array_map(function ($argument) {
    return ProcessUtils::escapeArgument($argument);
}, $this->getValues());

$commandline = implode(' ', $commandlineArgs);
```

or rather a simple **`foreach` loop** (for readability):

```
$commandlineArgs = [];
foreach ($this->getValues() as $argument) {
    $commandlineArgs[] = ProcessUtils::escapeArgument($argument);
}
```